src/crawlers.js

```diff
@@ -111,6 +193,26 @@ export default {
         return requestPromise(requestParams);
     },
 
+    getLastExecutionResults: (requestPromise, options) => {
```

Comment:
Same as `getListOfCrawlers` - we should encapsulate result in `{ items: [...] }` or something like that